server/screeningsList.js

```diff
@@ -0,0 +1,43 @@
+import fetch from 'node-fetch';
+
+export const getScreeningsWithMovies = {
+    loadData : async () => {
+        const res = await fetch("https://plankton-app-xhkom.ondigitalocean.app/api/screenings?populate=movie");
+        const data = await res.json();
+
+        return data;
+    }
+}
+// Formatting and filtering JSON to send only relevant data for rendering. 
+export const getScreeningsList = async (apiHandler) => {
+        
+    const data = await apiHandler.loadData();
+    
+    const res = data.data
+    
+    .map( array => ({
+        id: array.id, 
+        ...array.attributes,
+        movie: {
+            id: array.attributes.movie.data.id,
+            ...array.attributes.movie.data.attributes
+        }
+    }))
+     
+    .filter(screeningDate => {
```

Comment:
The parameter you've called `screeningDate` here is really the full screening object, i.e. not just a date. I would call it `screening`.